src/client.js

```diff
@@ -223,20 +183,8 @@ export function pullImage(system, reference) {
     });
 }
 
-export function pruneUnusedImages(system) {
-    return new Promise((resolve, reject) => {
-        podmanCall("libpod/images/prune?all=true", "POST", {}, system).then(resolve)
-                .then(reply => resolve(JSON.parse(reply)))
-                .catch(reject);
-    });
-}
+export const pruneUnusedImages = system => podmanJson("libpod/images/prune?all=true", "POST", {}, system);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1339-20230706-033702-672c013e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)